Mesh_3/include/CGAL/Mesh_vertex_base_3.h

```diff
@@ -66,6 +66,13 @@ template <>
 class Mesh_vertex_base_3_base<Parallel_tag>
 {
 public:
+  Mesh_vertex_base_3_base()
+  {}
+
+  Mesh_vertex_base_3_base( const Mesh_vertex_base_3_base& c)
+  {
+    m_erase_counter.store(c.erase_counter());
+  }
```

Comment:
I see that this copy-constructor is needed by Mesh_3 global optimizers. Where as the `m_erase_counter` is actually only required by the core of the meshing process, in parallel.

I really wish we had dynamic properties, in CGAL triangulations, like in `CGAL::Surface_mesh`!